### ISSUE TITLE
RemoteService Exception

### ISSUE THREAD

**Issue by manash-consultancy** [danielR2001/flutter_exoplayer#38]
Hi, crashlytics is showing a remote service exception. Roughly 6-7% users are affected. How can I resolve this?

![Screenshot 2020-04-28 at 8 14 27 PM](https://user-images.githubusercontent.com/18283444/80502650-88b2c500-898e-11ea-8b9f-529bb32b6f98.png)

**Comment by manash-consultancy** [comment]
First of all, the library is great @danielR2001 . Many thanks. 

In the production version of our app which is being used by thousands of users, two main crash clusters (related to this library) were observed - (1) Null pointer exceptions in ForegroundPlayer startcommand and (2) RemoteServiceExceptions like the one mentioned above. 

(1) were resolved by simply checking for null values, wherever possible and appropriately returning sticky & non_sticky in the startcommand function

(2) was a bit more complicated. We ended up over-riding the onCreate function and calling startForeground() from onCreate as well. 

Would love to contribute these changes to the library.
